packages/codebytes/src/types.ts

```diff
@@ -0,0 +1,13 @@
+import { LanguageOption } from './consts';
+
+type CodebytesChangeHandler = (text: string, language: LanguageOption) => void;
```

Comment:
We can change this type to an interface, based on the same approach we made earlier